messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistModalActivity.kt

```diff
@@ -135,23 +173,30 @@ class GistModalActivity : AppCompatActivity(), ModalInAppMessageViewCallback, Tr
             animationSet.start()
             animationSet.doOnEnd {
                 logger.debug("GistModelActivity finish animation completed")
-                super.finish()
+                completeFinish()
```

Comment:
Can the animation potentially be `cancelled` if the activity is destroyed before completion? 
Should we add animation cancellation in `onDestroy()`?